example/onetype.ts

```diff
@@ -0,0 +1,3 @@
+export default interface OneType {
+    someProp: number;
```

Comment:
Would maybe be even more educative to include nested types in the example? By changing this to: 
`someProp: AnotherType;`
WDYT?

Reply:
Would be a great addition! Added it